regtests/0239_priority/priorities.adb

```diff
@@ -68,12 +68,10 @@ procedure Priorities is
    Previous_Chipher : ASU.Unbounded_String;
 
    Ciphers : ASU.Unbounded_String :=
-               To_Unbounded_String
-                 (if GNUTLS
-                  then "NORMAL"
-                       & (if Net.SSL.Version > "GNUTLS 3.6.3"
-                          then ":-VERS-TLS1.3" else "")
-                  else "DEFAULT");
+               To_Unbounded_String (if GNUTLS then "NORMAL" else "DEFAULT");
+
+   No_TLS13 : constant ASU.Unbounded_String
```

Comment:
Just a formatting point, see Ciphers above, the := should be end of line and the To_Unbounded_String indented unto constant (n of constant).

Reply:
;= location fixed, It is for No_TLS13 actually. The Ciphers can't be constant because we modify it below.